napari/_commands/_processor.py

```diff
@@ -0,0 +1,37 @@
+"""ConsoleProcessor class for our IPython console.
+"""
+from ._utils import highlight
+from ._layers import LayerCommands
+
+HELP_STR = f"""
+{highlight("Available Commands:")}
+cmd.help
+cmd.layers
+cmd.levels(layer_id)
+"""
+
+
+class CommandProcessor:
+    """Command processor designed for interactive use in the IPython console.
+
+    Type "viewer.cmd.help" in Python for valid commands.
+    """
+
+    def __init__(self, layerlist):
+        self.layer_commands = LayerCommands(layerlist)
+
+    def __repr__(self):
+        return HELP_STR
+
+    @property
+    def help(self):
+        print(HELP_STR)
```

Comment:
You do want to be careful with side effect on property it may be triggerd on tab completions.